src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -9,7 +9,14 @@
      * <p>Пример: n = 8
      * Результат: 21</p>
      */
+
     public int getFibonacciNumber(int index) {
-        return 0;
+        if (index == 0) {
+            return 0;
+        } else if (index == 1) {
+            return 1;
+        } else {
```

Comment:
Do we need this else block here? Try removing it.

Reply:
No, we do not need it